ng-dialog/ng-dialog.d.ts

```diff
@@ -106,5 +112,9 @@ declare module angular.dialog {
 		 * Scope object that will be passed to dialog. If you use controller with separate $scope service this object will be passed to $scope.$parent param.
 		 */
 		scope?: ng.IScope;
+		/**
+		 * Any serializable data that you want to be stored in the controller's dialog scope.
+		 */
+		data?: string|Object|Array<any>;
```

Comment:
Can you use the shorthand array syntax instead e.g.

`any[]` rather than `Array<any>` since that's the general usage of it